packages/0x.js/src/contract_wrappers/contract_wrapper.ts

```diff
@@ -34,12 +36,14 @@ export class ContractWrapper {
     protected _web3Wrapper: Web3Wrapper;
     private _networkId: number;
     private _abiDecoder?: AbiDecoder;
-    private _blockAndLogStreamerIfExists: BlockAndLogStreamer|undefined;
+    private _blockAndLogStreamerIfExists: BlockAndLogStreamer | undefined;
```

Comment:
Are we leaving this for now? I'm ok with it. It is consistent with the styling in many other projects.